nginx.conf

```diff
@@ -115,6 +115,7 @@ http {
 	passenger_pool_idle_time 300;
     passenger_max_pool_size 10;
     passenger_min_instances 1;
+    passenger_log_level 3;
```

Comment:
Even though `passenger_log_level 3` is [default](https://www.phusionpassenger.com/library/config/nginx/reference/#passenger_log_level), having this configuration explicit seems helpful.  I wouldn't know off-hand about this option, and I can see it be useful to adjust when diagnosing an issue.

Reply:
For reference, I got this from here: https://stackoverflow.com/questions/59167793/passenger-not-logging-ruby-stdout-stderr

This example had set it explicitly to `2`, and I figured declaring it as `3` would help insulate us from other upstream changes.